modules/ROOT/pages/cookies.adoc

```diff
@@ -0,0 +1,158 @@
+= Configure Cookies
```

Comment:
@electronwill - Suggested heading - About Exchange Cookies

Reply:
Sounds good. I've updated the PR to use "About Exchange Cookies".